packages/module/src/Message/LinkMessage/LinkMessage.tsx

```diff
@@ -0,0 +1,33 @@
+// ============================================================================
+// Chatbot Main - Message - Content - Link
+// ============================================================================
+
+import React from 'react';
+import { ExtraProps } from 'react-markdown';
+import { Button, ButtonProps } from '@patternfly/react-core';
+import { ExternalLinkSquareAltIcon } from '@patternfly/react-icons';
+
+const LinkMessage = ({ children, ...props }: JSX.IntrinsicElements['a'] & ExtraProps) => {
```

Comment:
this is a really interesting props type. 

I think i would have expected it to be `({ children, ...props }: ButtonProps)` is there a reason that doesn't work here?

Reply:
There's a lot of incompatible types we get back from the third-party library. This ranges from type to more esoteric things like onCopy. What we get back from the library is effectively something like this:

```
 {
      href: 'https://www.patternfly.org/',
      node: {
        type: 'element',
        tagName: 'a',
        properties: { href: 'https://www.patternfly.org/' },
        children: [ [Object] ],
        position: { start: [Object], end: [Object] }
      }
    }
```

I had experimented with mapping the types more directly, but once I got past 10 things that were going to be TypeScript errors, I started looking into alternatives.